packages/generate/src/queries.ts

```diff
@@ -158,7 +165,33 @@ export function stringifyImports(imports: ImportMap) {
     .join("\n");
 }
 
-async function getMatches(root: string, schemaDir: string) {
+async function getMatches(
+  root: string,
+  schemaDir: string,
+  patterns?: string[],
+) {
+  if (patterns && patterns.length > 0) {
```

Comment:
Instead of switching between `globby` based matching and `walk` based matching, let's use `globby` for everything to simplify.

I think that should be as easy as detecting the empty `patterns` array and pushing `**/*.edgeql`?